<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LissCalc</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::InputMethodHint::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <widget class="QSlider" name="phase_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::InputMethodHint::ImhNone</set>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="phase_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::InputMethodHint::ImhNone</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecionar ângulo de fase da corrente (°)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecionar amplitude da tensão (V):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vamp_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::InputMethodHint::ImhNone</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="iamp_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::InputMethodHint::ImhNone</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecionar amplitude da corrente (A):</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="sine" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="liss" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="vector" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="freqList">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1:1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1:2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1:3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3:4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4:5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5:6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecionar relação entre frequências (V:A)</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Arquivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_File_Ctrl_N">
   <property name="text">
    <string>New File (Ctrl+N)</string>
   </property>
  </action>
  <action name="action1_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1:1</string>
   </property>
  </action>
  <action name="action2_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1:2</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1:3</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2:3</string>
   </property>
  </action>
  <action name="action3_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3:4</string>
   </property>
  </action>
  <action name="action4_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4:5</string>
   </property>
  </action>
  <action name="action5_6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5:6</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
